### ISSUE TITLE
Cannot install version 0.12.0

### ISSUE THREAD

**Issue by rwysocki-equinix** [Code-Hex/graphql-codegen-typescript-validation-schema#487]
When running `npm i graphql-codegen-typescript-validation-schema`

I get the following error message:
```
npm ERR! code 1
npm ERR! path [...]/node_modules/graphql-codegen-typescript-validation-schema
npm ERR! command failed
npm ERR! command sh -c -- npx only-allow pnpm
npm ERR! npm ERR! code SELF_SIGNED_CERT_IN_CHAIN
npm ERR! npm ERR! errno SELF_SIGNED_CERT_IN_CHAIN
```

Downgrading to version 0.11.1 fixes the issue.
I believe that line: `"preinstall": "npx only-allow pnpm",` should be used only in projects, not libraries like this one, as it also propagates to projects using it. I think the intention behind this line was to enforce that every contributor is using pnpm, not every end-user

**Comment by paales** [comment]
Running into this as well. @Code-Hex, ideas?

**Comment by Code-Hex** [comment]
@rwysocki-equinix @paales I'm sorry late. I was busy.

I just published the latest version as v0.12.1 which includes this fix
